### PR TITLE
Document changes from YUNIKORN-1087

### PR DESCRIPTION
**Pull request by anuraagnalluri**

### CODE REVIEW

**Comment by anuraagnalluri** [comment]
Thanks @craigcondit. Updated the PR to include a small explanation in the Overview to keep it DRY.

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -375,6 +377,9 @@ In the example below there are three allocations belonging to two applications.
                     "kubernetes.io/meta/namespace": "default",
                     "kubernetes.io/meta/podName": "task0"
                 },
+                "requestTime":1648754034098912461,
```
Comment (craigcondit) [inline docs/api/scheduler.md:380]: Let's add a space after these to be consistent with the rest of the file. There's several instances of this.

File: docs/api/scheduler.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ Many of these APIs return collections of resources. Internally, all resources ar
 are returned in units of bytes while resources of type `vcore` are returned in units of millicores
 (thousands of a core). All other resource types have no specific unit assigned.
 
+Under the `allocations` field in the response content for the apps/nodes related calls in the following spec, `placeholderUsed` refers to whether or not the allocation is a replacement for a placeholder. If true, `requestTime` is the creation time of its placeholder allocation, otherwise it's that of the allocation's ask. `allocationTime` is the creation time of the allocation, and `allocationDelay` is simply the difference between `allocationTime` and `requestTime`.
```
Comment (craigcondit) [inline docs/api/scheduler.md:34]: Let's make `apps/nodes` -> `app / node-related`.

**Review by craigcondit — CHANGES_REQUESTED** [review]
Looks good, other than some spacing / wording issues.

**Pull request closed without merging**